### PR TITLE
Hw8 andriy povzun

### PR DESCRIPTION
**Pull request by AndriyPovzun**

### CODE REVIEW

File: Andriy_Povzun/HW8_Andriy_Povzun/Task2.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from typing import Callable
+from typing import Iterable
```
Comment (bohdandovbsyh) [inline Andriy_Povzun/HW8_Andriy_Povzun/Task2.py]: from typing import Callable, Iterable

File: Andriy_Povzun/HW8_Andriy_Povzun/Task3.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from typing import Callable
+from typing import Iterable
```
Comment (bohdandovbsyh) [inline Andriy_Povzun/HW8_Andriy_Povzun/Task3.py]: same here

File: Andriy_Povzun/HW8_Andriy_Povzun/Task4.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
+def my_max(objects: list, amount_of_result=1):
+    max_number = objects[0]
+    second_max = objects[0]
+    if amount_of_result > 2 or amount_of_result < 1:
+        raise Exception("The maximum number of objects that can be returned is 2")
```
Comment (bohdandovbsyh) [inline Andriy_Povzun/HW8_Andriy_Povzun/Task4.py]: why so? I want more max numbers to be printed not 1 or 2 ))

File: Andriy_Povzun/HW8_Andriy_Povzun/Task4.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+"""
+An alternative to the max() built-in function
+"""
+
+
+def my_min(objects: list, amount_of_result=1):
+    min_number = objects[0]
+    second_min = objects[0]
+    if amount_of_result > 2 or amount_of_result < 1:
+        raise Exception("The maximum number of objects that can be returned is 2")
```
Comment (bohdandovbsyh) [inline Andriy_Povzun/HW8_Andriy_Povzun/Task4.py]: same here

File: Andriy_Povzun/HW8_Andriy_Povzun/Task2.py (hunk 1 above)
Comment (AndriyPovzun) [reply Andriy_Povzun/HW8_Andriy_Povzun/Task2.py]: Fixed

File: Andriy_Povzun/HW8_Andriy_Povzun/Task3.py (hunk 2 above)
Comment (AndriyPovzun) [reply Andriy_Povzun/HW8_Andriy_Povzun/Task3.py]: fixed

File: Andriy_Povzun/HW8_Andriy_Povzun/Task4.py (hunk 3 above)
Comment (AndriyPovzun) [reply Andriy_Povzun/HW8_Andriy_Povzun/Task4.py]: Okay, I'm fix it with your support, thanks)

File: Andriy_Povzun/HW8_Andriy_Povzun/Task4.py (hunk 4 above)
Comment (AndriyPovzun) [reply Andriy_Povzun/HW8_Andriy_Povzun/Task4.py]: done

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**